File: Data Structures & Algorithms/two-integer-sum/submission-0.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int i = 0;
        int j = 0;

        for (int k = 0; k < nums.length; k++) {
            i = nums[k];
            for (int l = k+1; l < nums.length; l++) {
                j = nums[l];
                if (i + j == target) return new int[]{k, l};
            }
        }
        return null; 
    }
}

File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.py
from typing import List

class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        point1, point2 = 0, len(numbers) - 1
        
        while point1 < point2:
            sumation = numbers[point1] + numbers[point2]
            if  sumation == target:
                return [point1 + 1 , point2 + 1]
            elif sumation > target:
                point2 = point2 - 1
            else:
                point1 = point1 + 1
        
        return None